packages/portal/src/components/item/ItemRemoveButton.vue

```diff
@@ -8,11 +8,20 @@
       data-qa="item remove button"
       :aria-label="$t('actions.remove')"
       :title="tooltipTitle"
-      @click="removeItem"
+      @click="handleClickButton"
     >
       <span class="icon-remove-circle-outlined" />
       {{ buttonText ? $t('actions.remove') : '' }}
     </b-button>
+    <ConfirmDangerModal
```

Comment:
On gallery pages, this results in many identical modals. They all have the same id attribute.
A side effect of that being that they are all shown.

We could declare the modal once pages that we know will potentially need it and then pass the ID as a ref to the buttons.

Reply:
I have refactored so that it's only rendered when it's needed.